nixos/modules/services/hardware/uinput.nix

```diff
@@ -0,0 +1,28 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.hardware.uinput;
+in {
+  options.services.hardware.uinput = {
```

Comment:
Should be just `options.hardware.uinput`. It's not a service after all

Reply:
Fixed.